src/app/content/hooks/locationChange/trackPageView.ts

```diff
@@ -0,0 +1,11 @@
+import googleAnalyticsClient from '../../../../gateways/googleAnalyticsClient';
+import { Match } from '../../../navigation/types';
+import { content } from '../../routes';
+
+import { assertWindow } from '../../../../app/utils';
+
+export default async(match: Match<typeof content>) => {
+  // match.route.getUrl(match.params) is the path, how to get the domain?
+  console.log(match); // to hide unused variable error
+  googleAnalyticsClient.trackPageView(assertWindow().location.href);
```

Comment:
tried to use `match` but couldn't figure out the domain.  is `window.location` good enough here?

Reply:
does it want the full domain in the url or is just the path enough? if you need everything `window.location` is probably fine, or we could add it to the `Navigation/locationChange` payload

Reply:
you are right, path is enough, so I'm using `locationChange.payload.location.pathname`